lightning/src/ln/functional_tests.rs

```diff
@@ -10688,7 +10789,12 @@ fn test_disconnect_in_funding_batch() {
 	// The remaining peer in the batch disconnects.
 	nodes[0].node.peer_disconnected(&nodes[2].node.get_our_node_id());
 
-	// The channels in the batch will close immediately.
+	// After the time expires for allowing peer to connect back
```

Comment:
This reads a bit incomplete. Let's rephrase this in similar comments elsewhere as:

```rust
// Let the time for the peer to reconnect expire.
```

Reply:
Thanks for the suggestion!
Addressed it in [f9fbdca](https://github.com/lightningdevkit/rust-lightning/commit/f9fbdcad5e018c8533fc3670bba3dd4b9fd281ce)